gremlin-go/README.md

```diff
@@ -117,47 +68,114 @@ require (
 ```
 As well as a populated `go.sum` file.
 
-This following example should run, provided that it is configured to point to a compatible `gremlin-server`. In this example, a simple local server is running on port 8182, and this will print`[2]` as an output. If no server is available, this code can still be executed to print an error as output.
+*if there are no usages for gremlingo found, it will remove the require from go.mod and not import dependencies.*
```

Comment:
We could also suggest doing the next step (of setting up an example file that uses gremlingo) and then running go mod tidy.

Reply:
I think we have this above don't we? @L0Lmaker @vkagamlyk